podcasts/FoldersCoordinator.swift

```diff
@@ -150,6 +151,18 @@ class FoldersCoordinator: NSObject {
         NotificationCenter.postOnMainThread(notification: ServerNotifications.podcastsRefreshed, object: nil)
     }
 
+    private func saveLastUuidsUsed() {
+        let uuids = dataManager.allPodcastsOrderedByAddedDate().map { $0.uuid }.sorted()
+        let md5 = String(uuids.joined(separator: "")).md5
```

Comment:
Can we avoid to repeat these 2?

Reply:
Silly question but does this mean we fetch all podcasts each time? Would the last podcast added be enough? I'm worried about users with huge database.

Reply:
There is a cache in the data manager, so we are not hitting the database and not using more memory than it's already being used.

Reply:
Thanks for addressing that! Im wondering if there's a better way to now if that column in that table is modified 🤔